extensions/graphite-emitter/README.md

```diff
@@ -0,0 +1,50 @@
+## introduction
+
+This extension emits some selected druid metrics to a graphite carbon server.
+Events are sent after been [pickled](http://graphite.readthedocs.org/en/latest/feeding-carbon.html#the-pickle-protocol) , the size of the batch is configurable.
+Since Graphite doesn't support alerting, alerts will be redirect to warning logs by default. 
+
+ Metrics are organized using the following schema:
+`<namespacePrefix>.[<druid service name>].[<druid hostname>].<druid metrics dimensions>.<druid metrics name>`
+Properly naming your metrics is critical to avoid conflicts, confusing data and potentially wrong interpretation later on.
+
+Example `druid.historical.hist-host1.yahoo.com:8080.MyDataSourceName.GroupBy.query/time`:
+
+ * `druid` -> namespace prefix 
+ * `historical` -> service name 
+ * `hist-host1.yahoo.com:8080` -> druid hostname
+ * `MyDataSourceName` -> dimension value 
+ * `GroupBy` -> dimension value
+ * `query/time` -> metric name
+
+User has control of `<namespace>.[<druid service name>].[<druid hostname>].`
+
+You can omit the hostname by setting `druid.emitter.graphite.ignoreHostname=true`
+`druid.SERVICE_NAME.dataSourceName.queryType.query.time`
+You can omit service name by setting `druid.emitter.graphite.ignoreServiceName=true`
+`druid.HOSTNAME.dataSourceName.queryType.query.time`
+## configuration
+To use graphite as emitter first set `druid.emitter=graphite`
+Also graphite can be used via the composing emitter.
+All the configuration for graphite emitter are under `druid.emitter.graphite`
+
+|property|description|required?|default|
+|--------|-----------|---------|-------|
+|`druid.emitter.graphite.hostname`|The hostname of the graphite server.|yes|none|
+|`druid.emitter.graphite.port`|The port of the graphite server.|yes|none|
+|`druid.emitter.graphite.batchSize`|Number of event to send as one batch.|no|100|
+|`druid.emitter.graphite.namespacePrefix`|The prefix of the namespace path. |no| druid|  
+|`druid.emitter.graphite.ignoreServiceName`| Strip the service name from the metric path. |no|`false`|
+|`druid.emitter.graphite.ignoreHostname`|Strip the hostname from the metric path.|no|`false`|
+|`druid.emitter.graphite.logAlerts`|Enable logging of alerts as WARN.|no|`true`|
+|`druid.emitter.graphite.flushPeriod` | Queue flashing period. |no|1 minute|
+|`druid.emitter.graphite.maxQueueSize`| Maximum size of the queue used to buffer events. |no|`MAX_INT`|
+
+
+## QuickStart
+If you want to experiment with this module you can quickly setup a graphite/graphana server using a docker container:
+
+ - 1 First clone this [Docker project](https://github.com/b-slim/docker-grafana-graphite)
+
+ - 2 execute `build` then `start` scripts (assuming docker is working)
+ 
```

Comment:
instead of module README, please put this in a the doc file in the docs/development folder and link in TOC as an experimental feature